### PR TITLE
use gradle wrapper in bench script

### PR DESCRIPTION
**Pull request by djodjoni**

### CODE REVIEW

**Comment by fabienrenaud** [comment]
Added gradlew to the project with gradle 3.3 and the _all_ distribution.

File: bench [hunk 1]
```diff
@@ -5,5 +5,5 @@ if [ -z $JVM_OPTIONS ]; then
   JVM_OPTIONS="-server -XX:+AggressiveOpts -Xms2G -Xmx2G"
 fi
 
-gradle -q shadowJar
+./gradlew -q shadowJar
```
Comment (djodjoni) [inline bench]: the main purpose of this PR

**Comment by djodjoni** [comment]
there are 2 things:
1) i didnt push 3.3 because it doesnt run with it
2) there is no use to put a wrapper if it is not used and this was the main purpose of this PR

File: bench (hunk 1 above)
Comment (fabienrenaud) [inline bench]: done

Comment (djodjoni) [inline bench]: :+1:

**Pull request closed without merging**